templates/dashboard.php

```diff
@@ -1 +1,7 @@
-<?= sprintf(__('Total count of subscriptions: %d'), $settings['email_count']); ?>
+<?= sprintf(__('Total count of subscriptions: %d'), $settings['email_count']);
+print('<br /><br /><div style="height: 200px; overflow-y: scroll;"><table style="width: 100%;"><thead style="font-weight: bold;"><tr><th>Email address</th><th>Since</th></tr></thead>');
```

Comment:
I would love to see those styles in the *.css file dedicated to the dashboard panel, wdyt?

Reply:
That may be better, especially if you think there'll be more style elements further down the line. I don't have experience with using external CSS in WordPress admin panel widgets though, so I'll leave this with you if you'd like to implement it.

Reply:
Yes, let me rework your PR a bit. I will push new commits later today...

Anyways this really nice improvement. Feel invited to add more :)